W/Woff2/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Woff2"
+version = v"1.0.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/google/woff2/archive/v$(version)/woff2-$(version).tar.gz", "add272bb09e6384a4833ffca4896350fdb16e0ca22df68c0384773c67a175594")
```

Comment:
Actually, I don't know why this didn't error, but I'm not sure we can use this URL because I think it's unstable: https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/.  You should use a `GitSource` instead.

Reply:
Alright, I'll go ahead and fix that when I have a computer later today.

Reply:
Should be fixed by the new commit